### PR TITLE
[NU-1520] Workaround for handle multiple instances of questDB

### PR DESCRIPTION
**Pull request by lukasz-bigorajski**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala [hunk 1]
```diff
@@ -181,7 +182,9 @@ object QuestDbFEStatisticsRepository extends LazyLogging {
   }
 
   private def createRootDirIfNotExists(): File = {
-    val nuDir = File.temp.createChild("nu", asDirectory = true)
+    val nuDir = File.temp
+      .createChild("nu", asDirectory = true)
+      .createChild(UUID.randomUUID().toString, asDirectory = true)
```
Comment (arkadius) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala:187]: So after a restart, we will lose data?

Comment (lukasz-bigorajski) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala:187]: Yes, it's workaround to work with multiple instances. If it's not critical we can close PR.

Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala:187]: Ok, so as a temporary workaround it is fine. Let's only remember to fix it before the release. Maybe some jira task with fix version would help with that?

**Review by arkadius — APPROVED** [review]

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala (hunk 1 above)
Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala:187]: It is not critical IMO, we can leave for a while with this. There is an easy workaround for that - we just need to stop designer during tests and resume it after tests are finished

Comment (lukasz-bigorajski) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/QuestDbFEStatisticsRepository.scala:187]: Ok, so I close PR

**Pull request closed without merging**